hw/top_darjeeling/templates/toplevel.sv.tpl

```diff
@@ -543,10 +552,19 @@ slice = str(alert_idx+w-1) + ":" + str(alert_idx)
       .${k} (${v}),
     % endfor
     % for port, reset in m["reset_connections"].items():
-      % if lib.is_shadowed_port(block, port):
-      .${lib.shadow_name(port)} (${lib.get_reset_path(top, reset, True)}),
-      % endif:
-      .${port} (${lib.get_reset_path(top, reset)})${"," if not loop.last else ""}
+<%
+      is_shadowed_port = lib.is_shadowed_port(block, port)
+      if is_unmanaged_reset(top, reset['name']):
+        reset_port = top['unmanaged_resets'].get(reset['name']).signal_name
```

Comment:
Nit: maybe add support for unmanaged resets in topgen.lib, as in lib.get_reset_path?